mu-plugins/blocks/ratings-stars/render.php

```diff
@@ -30,11 +30,11 @@
 	<div class="wporg-ratings-stars__icons">
 		<?php
 
-		$display_rating = round( $data['rating'] / 10 ) * 0.5;
+		$display_rating = $data['rating'] / 10 * 0.5;
 		for ( $i = 0; $i < 5; $i++ ) {
 			if ( $i + 1 <= $display_rating ) {
 				echo '<svg class="is-star-filled" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24" aria-hidden="true" focusable="false"><path d="M11.776 4.454a.25.25 0 01.448 0l2.069 4.192a.25.25 0 00.188.137l4.626.672a.25.25 0 01.139.426l-3.348 3.263a.25.25 0 00-.072.222l.79 4.607a.25.25 0 01-.362.263l-4.138-2.175a.25.25 0 00-.232 0l-4.138 2.175a.25.25 0 01-.363-.263l.79-4.607a.25.25 0 00-.071-.222L4.754 9.881a.25.25 0 01.139-.426l4.626-.672a.25.25 0 00.188-.137l2.069-4.192z"></path></svg>';
-			} else if ( $i + 0.5 === $display_rating ) {
+			} else if ( $i + 0.5 <= $display_rating && $i + 1 > $display_rating ) {
```

Comment:
Do we need to check `$i + 1 > $display_rating` here? 
The first if condition checks `if ($i + 1) <= $display_rating`. If this condition fails, it implies that `$i + 1 > $display_rating` is already true.

Reply:
Yes! (AI is nice, but I'm falling out of love with it)